doc/devel/triage.rst

```diff
@@ -11,24 +12,28 @@ identifying major projects to work on, and discussing priorities.  For
 this reason, it is important to curate the issue list, adding labels
 to issues and closing issues that are resolved or unresolvable.
 
+Improving issues increases their chances of being successfully resolved.
```

Comment:
I find "Improving issues" somewhat confusing, because it can have one of the two meanings:
- work towards solving the issue (= improving an issue of the library)
- work towards improving the issue description.

Would "Improving issue reports" be ok?

Take or leave as this has been there before. We want to take quick improvements along the way, but don't want to get lost in details.